crates/aggchain-proof-builder/src/lib.rs

```diff
@@ -139,6 +139,9 @@ pub struct AggchainProofBuilder<ContractsClient> {
 
     /// Verification key for the aggchain proof.
     aggchain_vkey: Arc<SP1VerifyingKey>,
+
+    /// Static call caller address.
+    static_call_caller_address: Address,
```

Comment:
@hadjiszs Do we need this here, or we can keep it just in the `ContractsClient`?

Reply:
EDIT: I see it is used in the bridge witness. Not to change API of the `ContractsClient` with getter, better to leave it here.